include/FreeRTOS.h

```diff
@@ -2686,6 +2686,10 @@
     #endif
 #endif
 
+#ifndef configSTATS_BUFFER_MAX_LENGTH
+    #define configSTATS_BUFFER_MAX_LENGTH    0xFFFF
```

Comment:
This config now has effect on `vTaskList` and `vTaskGetRunTimeStats`. Suggesting we add the description in the task.h to indicate this.

Reply:
Updated.